printManual.c

```diff
@@ -0,0 +1,13 @@
+#include<stdio.h>
+#include "common.h"
+
+static const int MAX_COLORPAIR_NAME_CHARS = 16;
+const int MAX_COLOR_PAIRS = 25;
+void printColorCodingManual(){
```

Comment:
Good decision to place the new functionality in its own file 👍 
Reduces interference with existing / legacy code.

Reply:
Thank you very much Sudeep